group_vars/gitea.yml

```diff
@@ -0,0 +1,124 @@
+---
+firewalld_extra_allow_ports:
+  - "22/tcp"
+  - "80/tcp"
+  - "443/tcp"
+
+gitea_version: 1.21.11
```

Comment:
The latest release is `v1.22.1`. It looks like we're far behind on updates, including security ones. We should update this.

Reply:
Been holding on 1.21 branch because that's the last semblance of similarity between Gitea and Forgejo.  With the future still unknown we should hold out on 1.21 as long as possible IMO.

Right now there are no concerning security vulns in the 1.21 branch.